### PR TITLE
Updated onboarding text

### PR DESCRIPTION
**Pull request by cwarnermm**
#### Summary
Clarified onboarding text for new users.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-46606

#### Checklist
- [ ] Added or updated unit tests (required for all new features)
- [ ] Has UI changes
- [X] Includes text changes and localization file updates
- [ ] Have tested against the 5 core themes to ensure consistency between them.

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by enahum — CHANGES_REQUESTED** [review]

**Review by enahum — APPROVED** [review]

**Review by saturninoabril — APPROVED** [review]
Thanks Carrie! LGTM.

File: assets/base/i18n/en.json [hunk 1]
```diff
@@ -475,7 +475,7 @@
   "mobile.components.select_server_view.displayName": "Display Name",
   "mobile.components.select_server_view.enterServerUrl": "Enter Server URL",
   "mobile.components.select_server_view.msg_connect": "Let’s Connect to a Server",
-  "mobile.components.select_server_view.msg_description": "A Server is your team's communication hub which is accessed through a unique URL",
+  "mobile.components.select_server_view.msg_description": "A server is your team's communication hub accessed using a unique URL",
```
[the remark this answers is not in the document]
Comment (saturninoabril) [reply assets/base/i18n/en.json:478]: Hello @cwarnermm, I just noticed from the ticket that the new description has period (.). Might not be necessary but mentioning here.

Feel free to merge anytime.

[the remark this answers is not in the document]
Comment (cwarnermm) [reply assets/base/i18n/en.json:478]: Thanks, @saturninoabril! You have a great eye for detail :) I intentionally omitted the period feeling that it wouldn't add value based on how this text displays in-product.

**Comment by cwarnermm** [comment]
@enahum - Feel free to merge this PR whenever it's convenient for you.

**Pull request merged**